### DISCUSSION TITLE
It can not work in Kylin v10

### DISCUSSION THREAD

**Discussion by zhangluqi** [amwx/FluentAvalonia#234] [Q&A] [answered]
I run this demo in Kylin, it said default font is null or empty
I tried specifying the font, but it didn't work

**Comment by amwx** [comment, marked as answer]
How did you specify the font? It sounds like Skia isn't able to find a default font from your OS, so you need to set one in the AppBuilder setup in program.cs

I think it's something like (this is from memory so it may be wrong)
```C#
.With(new FontManagerOptions { DefaultFontFamily = new FontFamily("Your font") })
```

**Reply by zhangluqi** [reply]
I got font from Kylin, and add in FluentAvalonia.ll. then Specify the font in the ResolveLinuxSystemSettings method

`AddOrUpdateSystemResource("SymbolThemeFontFamily", new FontFamily(new Uri("avares://FluentAvalonia"), "/Fonts/#My font name"));`

it works fine in window, but not under Linux。
I really like your code and am learning it

**Reply by amwx** [reply]
Don't set the font that way. For one, that line is referring to the symbol font I use, which is embedded in the assembly. As I said above, set the default font using FontManagerOptions in the AppBuilder:
```C#
public static AppBuilder BuildAvaloniaApp()
			=> AppBuilder.Configure<App>()
				.UsePlatformDetect()
				.LogToTrace()
				.With(new FontManagerOptions
				{
					DefaultFamilyName = "Arial"
				});
```

But replace "Arial" with the font you want to use. This will tell Avalonia's font manager what to use when `FontFamily.Default` is called.

**Reply by zhangluqi** [reply]
thanks, it`s work well!!
